src/App.tsx

```diff
@@ -1,14 +1,83 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [filter, setFilter] = useState('all');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [activeTodo, setActiveTodo] = useState<Todo | undefined>(undefined);
```

Comment:
```suggestion
  const [activeTodo, setActiveTodo] = useState<Todo | null>(null);
```
Not critical, but it's more precise - we know that active todo is just not selected

Reply:
Done, that should make more sense now.